server/api/controllers/liveSermon/scheduler-live-sermon.js

```diff
@@ -0,0 +1,60 @@
+const cron  = require('node-cron');
+
+// To change the isPublished value in the database from false to true
+const onLiveButon = async () => {
+  try{
+    var mongoClient = LiveSermon.getDatastore().manager.client;
+    var results = await mongoClient.db('hmcchk_db_test')
+    .collection('livesermon');
+    //console.log("isPublished button has been changed")
+    var checking = await mongoClient.db('hmcchk_db_test')
+    .collection('livesermon')
+    .find({}, { isPublished: 1})
+    .toArray();
+    //console.log(checking);
+    if (!checking[0].isPublished) {
+      var data = await results.updateOne(
+        { isPublished: false },
+        { $set: { isPublished: true } },
+      )
+    }
+  }catch(err){
+    console.log(err);
+  }
+}
+
+// To change the isPublished value in the database from true to false
+const offLiveButton = async () => {
```

Comment:
onLiveButton and offLiveButton has a lot of code overlap, could we make change it to a function swtichLiveButton(off/on) instead